pyonfx/ass_core.py

```diff
@@ -486,6 +504,17 @@ def get_media_abs_path(mediafile):
                     self.meta.video = get_media_abs_path(line[11:].strip())
                     line = "Video File: %s\n" % self.meta.video
 
+                    if os.path.isfile(self.meta.video):
+                        self.input_timestamps = VideoTimestamps.from_video_file(
+                            Path(self.meta.video), video_index
+                        )
+                    else:
+                        fps_str = line[11:].strip().split(":")[1]
```

Comment:
Why line 11? Doesn't appear to be safe for all the .ass files

Reply:
For the same reason as [this](https://github.com/moi15moi/PyonFX/blob/8e0b61a3479c49e6ab89bfc601c9ceea93dc5511/pyonfx/ass_core.py#L504), to skip ``Video File:``.
Also, this part is covered by the test [``test_ass_values``](https://github.com/CoffeeStraw/PyonFX/blob/8e0b61a3479c49e6ab89bfc601c9ceea93dc5511/tests/test_ass.py#L142-L145).

Reply:
Ops :)
The original code is so old I didn't remember.

Makes sense then! Let me have a final read and I'll get this through!